pilot/pkg/serviceregistry/kube/controller/ambient/services.go

```diff
@@ -147,6 +147,12 @@ func (a *index) constructServiceEntries(svc *networkingclient.ServiceEntry, w *W
 		})
 	}
 
+	var lb *workloadapi.LoadBalancing
+	preferClose := strings.EqualFold(svc.Annotations["networking.istio.io/traffic-distribution"], v1.ServiceTrafficDistributionPreferClose)
```

Comment:
should this be in the api repo

Reply:
yeah I think we can probably move it over once this is merged? No point getting the API approved if we don't approve of adding it at all